src/ron/server.go

```diff
@@ -766,9 +799,22 @@ func (s *Server) clientHandler(c *client) {
 		}
 	}
 
-	if err != io.EOF && !strings.Contains(err.Error(), "connection reset by peer") {
-		log.Errorln(err)
+	// This is an OK error - likely just means the TCP connection was closed.
+	if err == io.EOF {
+		return
 	}
+
+	// This is an OK error - likely just means the client closed the connection.
+	if strings.Contains(err.Error(), "connection reset by peer") {
+		return
```

Comment:
Lets log this at least as info or debug and we can approve

Reply:
Debug log added. :+1: